### ISSUE TITLE
Change name rom for may street fighter 2 game 

### ISSUE THREAD

**Issue by luizaoooobr** [finalburnneo/FBNeo#1716]
Hello fbneo devs, I created a street fighter 2 rom hack where I, with the sf2hfu rom, changed the speed to be the same as the street fighter 2 champion edition and I also created new colors and a new menu, but I can't adjust the name from the rom from sf2hfu to sf2srsf, can someone help me, I changed the d_cps1 to have the romset, the generator.h with the rom data, but it didn't help, can anyone help me?
I changed fbneo_libretro
but the invalid romset error always occurred.
crc code is :
static struct BurnRomInfo Sf2srsfRomDesc[] = {
    { "s2tu_23.8f",    0x080000, 0xcaf0c875, BRF_ESS | BRF_PRG | CPS1_68K_PROGRAM_NO_BYTESWAP },
    { "s2tu_22.7f",    0x080000, 0x20fff8c3, BRF_ESS | BRF_PRG | CPS1_68K_PROGRAM_NO_BYTESWAP },
    { "s2tu_21.6f",    0x080000, 0x325db237, BRF_ESS | BRF_PRG | CPS1_68K_PROGRAM_NO_BYTESWAP },

    { "s92-1m.3a",     0x080000, 0x03b0d852, BRF_GRA | CPS1_TILES },
    { "s92-3m.5a",     0x080000, 0x840289ec, BRF_GRA | CPS1_TILES },
    { "s92-2m.4a",     0x080000, 0xcdb5f027, BRF_GRA | CPS1_TILES },
    { "s92-4m.6a",     0x080000, 0xe2799472, BRF_GRA | CPS1_TILES },
    { "s92-5m.7a",     0x080000, 0x75e20f8c, BRF_GRA | CPS1_TILES },
    { "s92-7m.9a",     0x080000, 0xee803831, BRF_GRA | CPS1_TILES },
    { "s92-6m.8a",     0x080000, 0x779c37ed, BRF_GRA | CPS1_TILES },
    { "s92-8m.10a",    0x080000, 0x005867b6, BRF_GRA | CPS1_TILES },
    { "s92-10m.3c",    0x080000, 0xd374589b, BRF_GRA | CPS1_TILES },
    { "s92-12m.5c",    0x080000, 0xd43d5f31, BRF_GRA | CPS1_TILES },
    { "s92-11m.4c",    0x080000, 0x09e7ea4b, BRF_GRA | CPS1_TILES },
    { "s92-13m.6c",    0x080000, 0x136d9999, BRF_GRA | CPS1_TILES },

    { "s92_09.11a",    0x010000, 0x08f6b60e, BRF_PRG | CPS1_Z80_PROGRAM },

    { "s92_18.11c",    0x020000, 0x7f162009, BRF_SND | CPS1_OKIM6295_SAMPLES },
    { "s92_19.12c",    0x020000, 0xbeade53f, BRF_SND | CPS1_OKIM6295_SAMPLES },

    A_BOARD_PLDS

    { "s9263b.1a",     0x000117, 0x0a7ecfe0, BRF_OPT },    // b-board PLDs
    { "iob1.12d",      0x000117, 0x3abc0700, BRF_OPT },
    { "bprg1.11d",     0x000117, 0x31793da7, BRF_OPT },
    { "ioc1.ic7",      0x000104, 0xa399772d, BRF_OPT },    // c-board PLDs
    { "c632.ic1",      0x000117, 0x0fbd9270, BRF_OPT },
};

STD_ROM_PICK(Sf2srsf)
STD_ROM_FN(Sf2srsf)

**Comment by Igor-Arabe** [comment]
Eu tentei contacta lo desde 2023 que vc citou esse hack, mas vc nunca leu as mensagems o sei lá o que... entre em contato via discord

**Comment by taoenwen** [comment]
@luizaoooobr Do you mean you need to rename (d_cps1) sf2hfsce to sf2srsf?

**Comment by Igor-Arabe** [comment]
Nope. he asked to change sf2hfu to sf2srsf originally. I said this is impossible and after contact via discord, i added hack with 'sf2hfsce' as zipfileName.
So just close this 'issue'. thanks.

**Comment by taoenwen** [comment]
> Nope. he asked to change sf2hfu to sf2srsf originally. I said this is impossible and after contact via discord, i added hack with 'sf2hfsce' as zipfileName. So just close this 'issue'. thanks.

I see，thanks！
